src/domain/occurrence/eventOccurrenceForm/EventOccurrenceForm.tsx

```diff
@@ -112,6 +115,16 @@ const EventOccurrenceForm: React.FC<Props & GoToPublishingProps> = ({
       );
   }, [t]);
 
+  const validationSchema = React.useMemo(
+    () =>
+      testDateWithEnrolmentValidationSchema(
+        ValidationSchema.fields.date as DateSchema,
+        event?.pEvent,
+        ValidationSchema
```

Comment:
Hmm is there any point passing validationSchema as a parameter? Would it be better to just use it in the function, they are in the same file?

Reply:
I agree there isn't, but I just extend the default validation schema as hard coded, I don't see there much difference to the first version where the date field dependencies were part of the default schema. We use this schema only once and then the dependencies are always needed. Now I have just splitted it to 2 schemas where 1 is extending the another and only the extended one is ever used.

Reply:
If the extended one is only used the there shouldn't be any need fo extension :)

Reply:
or is there any chance that either  enrolmentStart or  enrolmentEndDays is null/undefined?

Reply:
> If the extended one is only used the there shouldn't be any need fo extension :)

Well, the first version of PR didn't have one...

> or is there any chance that either enrolmentStart or enrolmentEndDays is null/undefined?

they should always be filled, because every occurrence needs an event and they are mandatory properties for an event.

Reply:
> Well, the first version of PR didn't have one...
The problem was that we have values in forms initialValues that are not actually in the form but only used in validation which I fell is little bit confusing but maybe in the end it is matter of taste